COMPILING.md

```diff
@@ -0,0 +1,139 @@
+Jamulus can be compiled for Linux, Windows and macOS as follows. Note also the contributed [installation scripts](https://github.com/jamulussoftware/installscripts) for use unattended installs. There are also reports from people who successfully compile and run Jamulus on BSDs.
+Android and iOS are not officially supported, therefore there are no instructions yet.
```

Comment:
Android build can be seen in the automation scripts, of course.

Indeed, all the build instructions for all supported platforms could be inferred from the automation scripts...